### PR TITLE
Proposal for addition to the RISC-V WG team

### PR DESCRIPTION
**Pull request by dkhayes117**
Hi, I'd like to join the risc-v working group team!  I've created this pull request to add my name to the README.md for voting.  Thank you for the consideration.
Daniel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -278,6 +278,7 @@ The RISC-V team develops and maintains the core of the RISC-V crate ecosystem.
 - [@almindor]
 - [@Disasm]
 - [@thejpster]
+- [@dkhayes117](https://github.com/dkhayes117)
```
Comment (therealprof) [inline README.md:281]: We prefer to have all those links in a single place at the end of this file.

**Comment by therealprof** [comment]
👋 @rust-embedded/riscv team, @dkhayes117 is requesting to join the team which is pending your majority approval:

- [ ] @almindor 
- [ ] @Disasm 
- [ ] @thejpster

File: README.md (hunk 1 above)
Comment (dkhayes117) [reply README.md]: Ah, I see.  Fixed, thanks

**Review by almindor — APPROVED** [review]

**Review by Disasm — APPROVED** [review]

**Review by thejpster — APPROVED** [review]

**Review by burrbull — APPROVED** [review]

**Comment by therealprof** [comment]
Thanks @rust-embedded/riscv. I'll implement the decision hopefully later today.

**Review by therealprof — APPROVED** [review]
@dkhayes117 Welcome to the WG. 🎉

bors r+

**Pull request merged**